sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.query.sql

```diff
@@ -53,6 +54,7 @@ _previous AS (
     submission_date = DATE_SUB(@submission_date, INTERVAL 1 DAY)
     -- Filter out rows from yesterday that have now fallen outside the 28-day window.
     AND udf.shift_28_bits_one_day(days_seen_bits) > 0
+    AND sample_id IS NOT NULL
```

Comment:
Some additional discussion and an example query illustrating the problem is also here: https://mozilla.slack.com/archives/C4D5ZA91B/p1646085773661049